etl/steps/data/garden/hyde/2017/baseline.countries.json

```diff
@@ -154,7 +154,6 @@
   "San Marino": "San Marino",
   "Saudi Arabia": "Saudi Arabia",
   "Senegal": "Senegal",
-  "Serbia": "Serbia",
```

Comment:
There was no `Serbia` in the dataset and it resulted in non-unique index (it's excluded here to avoid warning).

> Didn't catch this in the past, but I wonder if mapping Serbia and Montenegro to Serbia is accurate.

Not sure either, Pablo has tons of code around historical regions, so maybe we should ask him.

Reply:
Let's proceed like this for now, I'll take a look later when I work on population!